templates/landing/homepage.html

```diff
@@ -0,0 +1,62 @@
+{% extends 'landing/app_layout.html' %}
+{% load static %}
+{% block content %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>AcadeMe</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="{% static 'css/homePage.css' %}" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+  </head>
+  <body>
+    <div class="d-flex flex-column justify-content-center">
+        <div class="top-container align-items-center d-flex flex-column justify-content-center align-items-center">
+          <div class="mt-5 mb-3">
+            <h1 class="font-weight-bold">Welcome To</h1>
+          </div>
+          <div>
+            <img class="logo mb-5" src="{% static 'img/academe_logo.png' %}" />
+          </div>
+          
+        </div>
+      </div>
+      <div class="d-flex flex-row bottom-container justify-content-center">
+        <div class="flex-column m-5">
+          <h3 class="my-2">Choose University</h3>
+          <div>
+            <select aria-label="choose-university" id="selectContent">
+              <option selected>Choose University</option>
```

Comment:
this landing page is submitted 3 times
We would expect to have dynamic option. 
Parse a list of universities and run for loop in tags,
In case you want to add Universities to the list, add in migration test files

Please review the the UI talk. of the blog post

Reply:
@Yarboa,
Me and @razashtar93 are working together to make the lust dynamic and sourced fro the DB

Reply:
Is it in this PR? or another different one? 
In case it is different please note it in PR description

Reply:
@Yarboa 
From what I underatand from the PR description, Dean is pushing this PR on the branch of Raz's PR that is already merged.

Reply:
yes, he could rebase, and see that this was not implemented,
In order not to miss it now,
Please rebase and fix
see this link
https://github.com/redhat-beyond/AcadeMe/pull/94/files#r767901359

Reply:
@Yarboa done